src/kasse.c

```diff
@@ -349,35 +314,23 @@ void buy_custom(void) {
   BYTE c = 0, i = 0;
   int negative = 1;
   char entered[5] = {'1', 0, 0, 0, 0};
-  char *input, name[20];
+  char name[MAX_ITEM_NAME_LENGTH + 1];
```

Comment:
I *think* this is incorrect: in buy_custom (“Freitext verkaufen”), we intentionally wanted to allow longer item names.

Reply:
"Freitext verkaufen" is planned to go away as it unnecessarily complicated binary logging. Adding items on the fly should be implemented at some point instead.